tests/Makefile

```diff
@@ -1,5 +1,5 @@
 CONTEXT =
-BUILD_IMAGE = nginx/nginx-ingress:edge
+BUILD_IMAGE = nginx/nginx-ingress:1.11.0
```

Comment:
I think the test files don't need to be changed, right?

Reply:
I don't see any difference. those test files are not used in pipelines and they don't affect the customer deliverable